src/components/draw/feature-edit-style.vue

```diff
@@ -27,17 +31,56 @@ const currentStyleComponent = computed(() =>
   feature?.featureType.replace('drawn', 'FeatureEditStyle')
 )
 
+function onColorSelect(colorEvent: Event) {
+  feature.featureStyle.color = (colorEvent.target as HTMLInputElement).value
+  feature.changed()
+}
+
+function onSizeChange(newSize: number) {
+  feature.featureStyle.size = newSize
+  feature.changed()
```

Comment:
I am not sure the component should be aware of the openlayers feature implementation, it would have been great to refactor and move all this in a service or maybe add new methods to our extended Feature class that already exists.

Reply:
Agreed, could be cleaner to wrap all this into the `DrawnFeature`...